ApiSamples/Samples/HighlightIndoorMapEntities.m

```diff
@@ -0,0 +1,127 @@
+#import "HighlightIndoorMapEntities.h"
+@import Wrld;
+
+@interface HighlightIndoorMapEntities ()
+
+@property (nonatomic) WRLDMapView *mapView;
+@property (nonatomic) NSArray<UIButton *> *buttons;
+
+@end
+
+@implementation HighlightIndoorMapEntities
+
+- (void)viewDidLoad
+{
+    [super viewDidLoad];
+
+    _mapView = [[WRLDMapView alloc] initWithFrame:self.view.bounds];
+
+    _mapView.autoresizingMask = UIViewAutoresizingFlexibleWidth | UIViewAutoresizingFlexibleHeight;
+
+    [_mapView setCenterCoordinate:CLLocationCoordinate2DMake(56.459966, -2.978167)
+                        zoomLevel:17
+                         animated:NO];
+
+    [self.view addSubview:_mapView];
+
+    NSMutableArray<UIButton *>* buttons = [NSMutableArray array];
+
+    [[self addButton: @"Go to floor" withIndex: 0 andSelector: @selector(goToFloor)] setEnabled:YES];
+    [buttons addObject: [self addButton: @"Highlight entities" withIndex: 1 andSelector: @selector(highlightEntities)]];
+    [buttons addObject: [self addButton: @"Clear highlights" withIndex: 2 andSelector: @selector(clearEntityHighlights)]];
```

Comment:
I know this is same functionality as corresponding wrld.js example ,but no way of user exiting the indoor map once entered.